templates/iot/common/020-ClusterRole-qdr-configurator.yaml

```diff
@@ -15,6 +15,7 @@ rules:
     - create
```

Comment:
```suggestion
```

Reply:
:+1: In addition I think the `create` operation can be removed. At least, according to the OLM manifest

Reply:
Yup, sounds good. Changed.